SingularityUI/app/models/TaskResourceUsage.coffee

```diff
@@ -6,4 +6,18 @@ class TaskResourceUsage extends Model
 
     initialize: ({ @taskId }) =>
 
-module.exports = TaskResourceUsage
+    # Calculate CPU Usage by comparing previous usage to current usage
+    setCpuUsage: ->
+        previous = @get('previousUsage')
+        currentTime = @get('cpusSystemTimeSecs') + @get('cpusUserTimeSecs')
+        previousTime = previous.cpusSystemTimeSecs + previous.cpusUserTimeSecs
+        timestampDiff = @get('timestamp') - previous.timestamp
+        cpus_used = (currentTime - previousTime) / timestampDiff
+        cpuUsageExceeding = (cpus_used / @get('cpusLimit')) > 1.10
+        
+        if cpuUsageExceeding
```

Comment:
what happens when the task transitions from running to finished, though? do we display the last valid resource usage, or remove it completely? (i'm literally not sure, need to double check)

need to decide what the proper action would be -- i'm personally leaning towards hiding resource usage